docs/ContentTypeParser.md

```diff
@@ -5,7 +5,7 @@ Natively, Fastify only supports `'application/json'` and `'text/plain'` content
 
 As with the other APIs, `addContentTypeParser` is encapsulated in the scope in which it is declared. This means that if you declare it in the root scope it will be available everywhere, while if you declare it inside a plugin it will be available only in that scope and its children.
 
-Fastify automatically adds the parsed request payload to the [Fastify request](./Request.md) object which you can access with `request.body`.
```

Comment:
Does it brake the navigation between the different pages in IDEs, GitHub, and similar stuff? I'm a bit concerned that those changes may break this.

Reply:
I assume you mean does changing from the `./Request.md` to `Request.md`.

There's already a page on `master` with this style.

The first link to Plugins on [this page](https://github.com/fastify/fastify/blob/master/docs/Decorators.md) uses this style of link, and doesn't 404 on GitHub or the Fastify site.

Reply:
Yeah, that was my question. 

I've checked it. Looks good to me, let's wait for others to review :)